phoenicis-library/src/main/java/org/phoenicis/library/LibraryConfiguration.java

```diff
@@ -37,13 +38,16 @@
 
     @Bean
     public LibraryManager libraryManager() {
-        return new LibraryManager(shortcutDirectory);
+        return new LibraryManager(shortcutDirectory, new ObjectMapper());
     }
 
     @Bean
     public ShortcutManager shortcutManager() {
-        return new ShortcutManager(shortcutDirectory, desktopShortcutDirectory, libraryManager(),
-                scriptsConfiguration.scriptInterpreter());
+        return new ShortcutManager(shortcutDirectory,
```

Comment:
As far as I remember we have a `Bean` somewhere for the `ObjectMapper`

Reply:
Where? There's one in `RepositoryConfiguration` but I don't think it makes sense to use that one. I followed the example of `EnginesConfiguration`.

Reply:
Do we have access to `PhoenicisGlobalConfiguration`?

Reply:
Yes. The `ContainersConfiguration` uses this.